backends/ubpf/ubpfParser.cpp

```diff
@@ -367,6 +394,16 @@ bool UBPFStateTranslationVisitor::preorder(const IR::MethodCallExpression* expre
                 compileExtract(expression->arguments->at(0)->expression);
                 return false;
             }
+
+            if (extMethod->method->name.name == p4lib.packetIn.advance.name) {
+                if (expression->arguments->size() != 1) {
+                    ::error("Exactly one argument expected %1%", expression);
```

Comment:
Use the version of error with an additional argument of type ErrType. Or perhaps call modelError if there is such a function.